CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [#2413](https://github.com/bbatsov/rubocop/issues/2413): Allow `%Q` for dynamic strings with double quotes inside them. ([@jonas054][])
 * [#2404](https://github.com/bbatsov/rubocop/issues/2404): `Style/Next` does not remove comments when auto-correcting. ([@lumeet][])
 * `Style/Next` handles auto-correction of nested offenses. ([@lumeet][])
+* `ProgressFormatter` correctly respects the `--no-color` switch. ([@savef][])
```

Comment:
I'd replace correctly with fully, as it was respecting the switch to some extent even before the fix.